spec/fixtures/modules/error/tasks/typed.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Are you planning on making this a cross platform task, or planning on restricting the integration tests to only ssh targets?

Reply:
I thin you could get away with having a single ruby task if you target `localhost`.